c/busybox-1.22.0/busybox_sv_comp-readlink.h

```diff
@@ -0,0 +1,16 @@
+ssize_t readlink(const char *path, char *buf, size_t bufsiz)
+{
+  (void)*path;
```

Comment:
It should serve two purposes:
1. Avoid compiler warnings about unused variables.
2. Using `*path` instead of just `path` it should force verifiers to perform a valid-deref check on that argument.